code/modules/antagonists/brother/brother.dm

```diff
@@ -77,22 +77,20 @@
 		flashed.balloon_alert(source, "[flashed.p_they()] resist!")
 		return
 
-	flashed.mind.add_antag_datum(/datum/antagonist/brother, team)
+	team.add_brother(flashed, key_name(source))
```

Comment:
yeah like if `add_brother` hits a `return FALSE` here for any reason, we should `return` out of this proc.